src/SFA.DAS.Tools.Support.Web/Views/Error/AccessDenied.cshtml

```diff
@@ -11,4 +11,10 @@
             You do not have access to this part of the service.
         </p>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+@foreach (var item in @User.Claims.ToList())
+{
+    <div> @item.Type</div> 
+    <div> @item.Value </div>
+}
```

Comment:
This will be removed prior to it going into production?

Reply:
it will be deleted once production is working

Reply:
removed